examples/clone/clone.bal

```diff
@@ -36,34 +35,31 @@ public function main() returns (Person, Person, string)? {
         address: address
     };
 
-    // Assign the `person` value to `anyValue`. So the type of the value has to be determined in runtime.
+    // Assign the `person` value to an `any` typed variable `anyValue`.
+    // The type of the value held by `anyValue` will be determined at run time.
     any anyValue = person;
 
-    // Invoke the clone builtin method. The return type is either `Person` or `error`. Note that the `error` is returned
-    // if the value being cloned is not of `anydata` type. If the type of the value being cloned can be determined in
-    // compile time, return value is exactly of the type of the value being cloned.
-    // e.g. Person result = person.clone();
-    // In above example, 'person' is cloned instead of 'anyValue'. In compile time itself, we know the return type
-    // should be 'Person'
+    // Invoke the `.clone()` built-in method. The return type is either `Person` or `error`. Note that `error` could be 
+    // returned if the variable on which `.clone()` is called is not of type `anydata`.
+    // If the type of the value being cloned can be determined at compile time and is `anydata`, the type of the
+    // return value is exactly the type of the value being cloned.
+    // e.g. `Person result = person.clone();`
+    // If `person` is cloned instead of `anyValue`, it would be known at compile time that the return type should be
```

Comment:
Change to "If the type `Person` is cloned instead of the type `any`, at compile time the compiler will determine that the return type is 'Person' type."

Reply:
Will change to
```suggestion
    // If `person` is cloned instead of `anyValue`, at compile time the compiler will determine that the return type is
```

IMO it's better to leave the variables instead of the types since what we are cloning is the value.